Audio.EndPoint.Controller.Wrapper/Audio.EndPoint.Controller.Wrapper.vcxproj

```diff
@@ -31,27 +31,27 @@
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>AudioEndPointControllerWrapper</RootNamespace>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
```

Comment:
The target need to stay windows 8.1 else it removes support for Windows 7 and 8. Windows 10 toolset is not backward compatible.

https://stackoverflow.com/questions/34706246/visual-studio-2015-change-target-platform-to-windows-7

Same for the other files. Only change the Platform toolset.